Xwt.XamMac/Xwt.Mac/EmbedNativeWidgetBackend.cs

```diff
@@ -1,30 +1,37 @@
-using AppKit;
+﻿using AppKit;
 using Xwt.Backends;
 
 namespace Xwt.Mac
 {
 	public class EmbedNativeWidgetBackend : ViewBackend, IEmbeddedWidgetBackend
 	{
 		NSView innerView;
+		bool reparent;
 
 		public EmbedNativeWidgetBackend ()
 		{
 
 		}
 
+		public NSView InnerView {
```

Comment:
Why is it necessary to acces the inner widget on XamMac backend level? Can't `Widget.Subview` be used to access the inner widget?